src/bin/cargo-mir-json.rs

```diff
@@ -23,18 +26,14 @@ fn show_help() {
     println!("{}", CARGO_MIR_JSON_HELP);
 }
 
-fn show_version() {
-    println!("{}", env!("CARGO_PKG_VERSION"));
-}
-
 fn main() {
     // Check for version and help flags even when invoked as 'cargo-mir-json'
     if env::args().any(|a| a == "--help" || a == "-h") {
         show_help();
         return;
     }
-    if env::args().any(|a| a == "--version" || a == "-V") {
-        show_version();
+    if version::has_flag(&mut env::args()) {
+        println!("{}", version::string());
```

Comment:
Since this `println!` invocation is repeated a lot, it may be worth factoring this out into its own `show_version()` version, similar to the already-existing `show_help()` function (as seen above).